src/mpc-hc/CDarkTheme.cpp

```diff
@@ -396,3 +484,17 @@ bool CDarkTheme::canUseWin10DarkTheme() {
 }
 
 
+CSize CDarkTheme::GetAveCharSize(HDC dc) {

```

Comment:
Unused function?

Reply:
Another DPI experiment to calculate the dialog units.  For now, unused, but there were some thoughts of setting the checkbox spacing to match the dialog units from win32 specifications.